### DISCUSSION TITLE
Exports Configs on Librouteros not working

### DISCUSSION THREAD

**Discussion by ghost** [luqasz/librouteros#196] [Q&A]
Hi everyone. I have a problem when i try: 

`from librouteros.login import plain
from librouteros import connect

method = plain

api = connect(
	username="erick",
	password="",
	host="",
	login_method=method
)

system = api.path('system', 'export')
export = api(cmd=str(system))
for a in export:
	print(a)`

It don't show me at screen the output, i only see nothing blank. I need help with it, Sorry my english is a little bad ✌🏼

**Comment by luqasz** [comment]
`/export` doesn't work through API. You can redirect output to file then read file contents.

**Comment by ghost** [comment]
Ok, I get it, thanks
